<launch>
    <node pkg="project_lidar_on_image" type="project_lidar_on_image_node" name="project_lidar_on_image_node" output="screen">
        <param name="calibration_file" type="string" value="$(find project_lidar_on_image)/data/zed12407.ini"/>
        <param name="transformation_matrix" type="string" value="$(find project_lidar_on_image)/data/transformation_mat.txt"/>
        <remap from="image_raw" to="zed/zed_node/left_raw/image_raw_color"/>
    </node>
</launch>
